curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60facde2d0dc61085b41063f.md

```diff
@@ -9,11 +9,11 @@ dashedName: step-33
 
 The `textarea` element acts like an `input` element of type `text`, but comes with the added benefit of being able to receive multi-line text, and an initial number of text rows and columns.
 
-To allow users to register with a bio, add a `label` with the text `Provide a bio:` followed by a `textarea` element, which requires a closing tag.
+To allow users to register with a bio, at the end of the `fieldset` add a `label` with the text `Provide a bio:` followed by a `textarea` element, which requires a closing tag.
```

Comment:
This is a monster of a sentence. I think we need simpler sentences. Perhaps:
```suggestion
Users will be able to register with a bio. Add a `label` with the text `Provide a bio:` at the end of the `fieldset`. Add a `textarea` element after the `label` element. Note that the `textarea` requires a closing tag.
```

Reply:
Sounds great! Your changes look a lot better.

Reply:
Sounds great! Your changes look a lot better.